app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert len(split_integer(8, 1)) == 1
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(6, 2) == [3, 3]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert len(split_integer(8, 1)) == 1
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert sorted(split_integer(17, 4)) == [4, 4, 4, 5]
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    assert len(split_integer(32, 6)) != 6
```

Comment:
The test `test_should_add_zeros_when_value_is_less_than_number_of_parts` does not correctly verify the expected behavior. According to the task description, when the value is less than the number of parts, the function should return parts that sum up to the value, potentially including zeros. Consider changing the assertion to check if the sum of the parts equals the original value, e.g., `sum(split_integer(32, 6)) == 32`.